src/functions/getValidatedCarNames.js

```diff
@@ -0,0 +1,19 @@
+import { PRINT_MESSAGES } from '../constants/messages.js';
+import { getUserInput } from '../utils/ioModule.js';
+import { splitByComma } from '../utils/stringUtils.js';
+import {
+  validateCarNames,
+  validateRacingCarInput,
+} from '../validator/validatePipeline.js';
+
+const getValidatedCarNames = async function getValidatedCarNamesFunc() {
+  const inputForRacingCars = await getUserInput(PRINT_MESSAGES.INPUT.CAR_NAME);
+  validateRacingCarInput(inputForRacingCars);
+
+  const carNamesArray = splitByComma(inputForRacingCars);
```

Comment:
```suggestion
  const carNamesArray = inputForRacingCars.split(",");
```
저도 이런 추상화를 했었는데 최종적으로 과한 추상화라고 결론을 짓게 되었는데 성희님은 어떻게 생각하시나요??

Reply:
의견 주셔서 감사합니다! 1주차 공통리뷰에 있던 동영상에도 비슷한 고민이 있었던 것으로 기억합니다. splitByComma 함수는 단순한 .split(",") 연산을 감싸는 형태이지만 제가 추상화가 필요한 작업이라고 생각한 이유는 splitByComma라는 이름을 통해 충분히 코드의 의도와 목적을 전달할 수 있으며 추가적으로 trim이 필요하다거나 하는 확장성 측면에서도 내부를 수정하면 되어 유지보수 측면에서도 유리하다고 생각했습니다! 이러한 이유로 저는 splitByComma가 과한 추상화보다는 유의미한 명확성을 더하는 추상화라고 생각했습니다. 추가 의견이 있으시면 알려주시면 감사하겠습니다! 😊

Reply:
맞아요! 추상화를 통해 확장에 유연하고, `splitByComma`와 같은 함수 이름을 통해 어떤 동작을 하는지 명확하게 나타낼 수 있는 점은 좋은 것 같아요!

하지만 말씀해주신 `trim()`을 하는 로직이 추가된다면 `splitByComma`라는 이름만으로는 `trim()`을 한다라는 동작을 유추할 수 없기 때문에 변경에 유연한 추상화는 아니였다는 생각이 드는 것 같아요🤔

Reply:
좋은 질문 감사합니다! 😊 앞서 설명드린 것처럼 splitByComma를 추상화한 이유는 코드의 의도를 더욱 명확히 전달하고 유지보수에 유리하게 하기 위함이었습니다. 특히 함수 이름을 통해 기능의 목적이 잘 드러나도록 하여 가독성을 높이려는 의도였습니다!

getRandomNum 함수는 미션 요구사항에서 0과 9라는 고정된 범위가 명시되어 있었기 때문에 인자를 사용하지 않고 상수로 고정해 불필요한 코드 복잡성을 줄이고자 했습니다. 명확한 고정값의 경우 상수로 관리하는 것이 코드의 의도를 직관적으로 드러내기 때문에 이해하기 더 쉽다고 판단했습니다. 요구사항이 바뀌지 않는다고 단정하기보다는 주어진 고정값은 상수로 관리하는 것이 바람직하다고 생각하여 상황에 따라 방식을 다르게 선택하게 되었습니다. 추가적인 의견이나 피드백이 있으시면 언제든 말씀해 주세요!